### ISSUE TITLE
Provide option to use dropdown select or checkboxes for taxonomy term selection

### ISSUE THREAD

**Issue by scott8035** [Crocoblock/suggestions#575]
It would be great if you could maintain your taxonomy terms for a CPT on the post edit screen using a list of the existing terms in either a dropdown or as checkboxes. Or in other words, instead of using the tag-style tax edit, use the category-style (for checkboxes), or a dropdown. We don't really want every user to have to guess which terms are out there.

There is an example of this technique at https://wordpress.stackexchange.com/a/229863/133061

**Comment by Floydtm** [comment]
@scott8035 Hi! Could you provide screenshots with examples. Thank You!

**Comment by scott8035** [comment]
Well no, I can't because it doesn't exist yet, it's a new feature request. And I did include an example of what it would look like in the link I provided. ???

**Comment by Floydtm** [comment]
Would this fit you? Enable this option http://prntscr.com/pl476j when creating a custom taxonomy. Then it will be displayed as checkboxes http://prntscr.com/pl47z0 http://prntscr.com/pl4892

**Comment by scott8035** [comment]
Yes! For the checkboxes at least. Any chance we could collapse a long list of them into a dropdown instead of checkboxes?
